### DISCUSSION TITLE
Can I change the config.json during training (after stopping it and using restore-path) ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1719] [General Q&A]
Hi,

Background : I am experiencing OOM during training always during the evaluation phase. So when it crashes I change some parameters (eg evaluation batch size) from the latest config.json and uses --config_path and --restore_path flags on the latest directory.

Now when it starts training it creates a new directory with a new config.json file and the changes I made previously are not taken into account. So for example if I set evaluation batch size to 4 in the config.json pointed by --config_path, when I open the newly created config.json I see an evaluation batch size of 8 (which corresponds to the value in the `train_vits.py` that is launched.

So my question is : can I change some parameters by providing a config.json with different parameter values than the ones that were set when the training initially started and then restore training with that modified config.json ? Or is it just not possible to change parameters ?

Thanks in advance

**Comment by erogol** [comment]
it is possible. Then use /bin/train_tts.py to continue from the updated config

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you @erogol, I was launching the recipe for vits instead of train_tts.py. I will give it a try.
